### PR TITLE
Fix/webview t and c

### PR DESCRIPTION
**Pull request by tjizep**

### CODE REVIEW

File: app/src/main/res/layout/activity_onboarding_web_view.xml [hunk 1]
```diff
@@ -18,5 +19,57 @@
...
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:theme="@style/AppTheme.AppBarOverlay"
+        android:background="@color/transparent"
+        android:layout_marginTop="7dp"
+        android:fitsSystemWindows="false"
+        android:layout_alignTop="@+id/activity_onboarding_web_view"
+        app:elevation="0dp">
+
+        <android.support.design.widget.CollapsingToolbarLayout
```
Comment (wscheffer) [inline app/src/main/res/layout/activity_onboarding_web_view.xml:33]: Why add CollapsingToolbarLayout?

File: app/src/main/java/org/gem/indo/dooit/views/onboarding/OnboardingWebActivity.java [hunk 2]
```diff
@@ -6,6 +6,8 @@
 import android.net.NetworkInfo;
```
Comment (wscheffer) [inline app/src/main/java/org/gem/indo/dooit/views/onboarding/OnboardingWebActivity.java:6]: Why do we have Two WebView Activities?

**Review by wscheffer — CHANGES_REQUESTED** [review]

File: app/src/staging/java/org/gem/indo/dooit/Constants.java [hunk 3]
```diff
@@ -8,8 +8,8 @@
 
     public static final Boolean DEBUG = true;
     //public static final String BASE_URL = "http://gem-bbb.seed.p16n.org";
-    public static final String BASE_URL = "http://ec2-52-19-36-136.eu-west-1.compute.amazonaws.com";
-    //public static final String BASE_URL = "http://10.0.2.2:8000";
+    //public static final String BASE_URL = "http://ec2-52-19-36-136.eu-west-1.compute.amazonaws.com";
+    public static final String BASE_URL = "http://10.0.2.2:8000";
```
Comment (wvictor-rr) [inline app/src/staging/java/org/gem/indo/dooit/Constants.java:12]: Please don't commit the `Constants` file pointing to the local dev server.

File: app/src/main/java/org/gem/indo/dooit/views/onboarding/OnboardingWebActivity.java (hunk 2 above)
Comment (wvictor-rr) [reply app/src/main/java/org/gem/indo/dooit/views/onboarding/OnboardingWebActivity.java:6]: This was my doing. Will be fixed in https://github.com/praekeltfoundation/gem-bbb-indo/pull/163

File: app/src/main/res/layout/activity_onboarding_web_view.xml (hunk 1 above)
Comment (wvictor-rr) [reply app/src/main/res/layout/activity_onboarding_web_view.xml:33]: We were trying something out in order to match the designs. We'll change this to a normal toolbar in https://github.com/praekeltfoundation/gem-bbb-indo/pull/163

**Pull request merged**